packages/wdio-cli/src/templates/snippets/cucumber.ejs

```diff
@@ -1,50 +1,29 @@
 <%  if (answers.framework === 'cucumber') { %>
-    // If you are using Cucumber you need to specify the location of your step definitions.
```

Comment:
If the intention of removing the comments is to make the configuration file more slim, then I would suggest also removing all values that are default values anyway. Wdyt?

Reply:
No it's not about that, it's an improvement on what we already have. See my comment to your question above this one